packages/mocha-runner/test/unit/mocha-test-runner.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import * as Mocha from 'mocha';
+import Mocha from 'mocha';
```

Comment:
Mocha? not mocha?

Reply:
It's a constructor function (or `class`) 🤷‍♂️